Shu/README.md

```diff
@@ -0,0 +1,27 @@
+## Xinyuan Shu Student Developer Setup Onboarding
+
+1.Cloned fork to local computer using ssh key
+
+2.Created the draft pull request
+
+3.Installed VS Code into the local computer
+
+4.Created "test.py" program
+
+6.Installed pre-commit,worked through 'pre-commit introduction'
+
+7.Added the pre-commit config file to PR.
+
+8.Installed Docker, worked through 'The Carpentries Incubator module on Docker'
+
+9.Created Dockerfile,push dockerfile to docker hub
+
+    to use this dockerfile, input commands: 
+
+    A) docker image pull xshu29/firstprecommit
+    
+    B) docker container run xshu29/firstprecommit
```

Comment:
```suggestion
    A) docker pull xshu29/firstprecommit:latest
    
    B) docker run xshu29/firstprecommit:latest
```

While it isn't wrong to use the [`container` CLI API](https://docs.docker.com/engine/reference/commandline/container/) to run containers, it is far more common to use the [`docker run` CLI API](https://docs.docker.com/engine/reference/commandline/run/) and so just for conforming to standards it is better to use it here. The `docker container` CLI is more common for controlling existing containers.

If `docker run` is executed without the `--rm` flag then each time it is run it will create a container and leave it until another process cleans it up explicitly. While there are advantages to this if you might want to reuse an existing container instance in the future for desktop use you don't want to create lots of copies that just pile up until you explicitly remove them with `docker stop` and `docker rm`. You can see all the containers that exist that haven't been removed with `docker ps -a`.

It is also a good idea to include the explicit image tag.